Documentation/user-manual.txt

```diff
@@ -1458,7 +1483,7 @@ If the diver moves, a trace of the route is obtained by saving a location every
 
 ===== Activate the automated recording of GPS locations
 
-The _Subsurface-mobile_ GPS menu has an option at the bottom labled
+The _Subsurface-mobile_ GPS menu has an option at the bottom balled
```

Comment:
I don't think that one really is 'balled' :-)

Reply:
the bane of using a spell checker. corrected